packages/core/src/checkout/checkout-service.ts

```diff
@@ -1384,6 +1384,21 @@ export default class CheckoutService {
         return this._dispatch(action, { queueId: 'extensions' });
     }
 
+    /**
+     * Renders an extension for a checkout extension region.
+     * Currently, only one extension is allowed per region.
+     *
+     * @alpha
+     * @param container The ID of a container which the extension should be inserted.
+     * @param region The name of an area where the extension should be presented.
+     * @returns A promise that resolves to the current state.
+     */
+    renderExtension(container: string, region: ExtensionRegion): Promise<CheckoutSelectors> {
+        const action = this._extensionActionCreator.renderExtension(container, region);
+
+        return this._dispatch(action, { queueId: 'renderExtension' });
```

Comment:
```suggestion
        return this._dispatch(action, { queueId: 'extensions' });
```

Reply:
`extensions` is already used as `loadExtensions` 's queueId.
Shall I remove or rename that one?

Reply:
What I meant is you can reuse the same queue ID. If you do that, you can ensure that these actions are executed sequentially to prevent possible race conditions.